pkg/network/network_test.go

```diff
@@ -88,7 +88,7 @@ var _ = Describe("Testing network", func() {
 
 	Describe("Render", func() {
 		Context("when given valid arguments", func() {
-			conf := &cnao.NetworkAddonsConfigSpec{ImagePullPolicy: v1.PullAlways, Multus: &cnao.Multus{}, LinuxBridge: &cnao.LinuxBridge{}}
+			conf := &cnao.NetworkAddonsConfigSpec{ImagePullPolicy: v1.PullAlways, Multus: &cnao.Multus{}, LinuxBridge: &cnao.LinuxBridge{}, PlacementConfiguration: &cnao.PlacementConfiguration{Workloads: &cnao.Placement{}}}
```

Comment:
Why do we need to add that? The conf should have `valid arguments`. It should be valid even without Placement

Reply:
it is a change I suggested so taking the liberty to answer :) 

the reason is that since we are using `LinuxBridge` then `renderLinuxBridge()` is called and that needs `PlacementConfiguration` to be configured or code will fail [here](https://github.com/kubevirt/cluster-network-addons-operator/pull/532/files#diff-05bcdfcb4d7a09161593a0118efd2a5dR33):
```
data.Data["Placement"] = conf.PlacementConfiguration.Workloads
```
btw this is the same for `ImagePullPolicy` [here](https://github.com/kubevirt/cluster-network-addons-operator/pull/532/files#diff-05bcdfcb4d7a09161593a0118efd2a5dR26)
 ``
data.Data["ImagePullPolicy"] = conf.ImagePullPolicy`
```

Reply:
Understood, closing my eyes